glean-core/ios/Glean/Glean.swift

```diff
@@ -110,6 +111,10 @@ public class Glean {
                 return
             }
 
+            if self.debugViewTag != nil {
```

Comment:
You can fix the cyclomatic complexity lint by either refactoring this function into more functions, or you can just disable it for init by putting the comment `// swiftlint:disable cyclomatic_complexity` before the function and `// swiftlint:enable cyclomatic_complexity` after the function.  This will only disable it for the one function.

Reply:
I will disable the lint, because right now this function looks very similar to Android and that simplifies development (at least for me).